08 Modules/06 Intra-package References.py

```diff
@@ -11,7 +11,8 @@
 
 
 
-from ..costumer import contact # We use the "." operator to navigate to the parent folders.
+from .ecommerce.costumer import contact # We use the "." operator to navigate to the parent folders.
+# "one dot ." navigate to the current folder
```

Comment:
Not sure if this is right

```suggestion
from ...costumer import contact # We use the ".." operator to navigate to the parent folders.
```